packages/tao/src/commands/run.ts

```diff
@@ -20,6 +20,10 @@ import * as chalk from 'chalk';
 import { logger } from '../shared/logger';
 import { NxJsonConfiguration } from '../shared/nx';
 import { splitTarget } from '../utils/split-target';
+import { readJsonFile } from '../utils/fileutils';
+import { buildTargetFromScript, PackageJson } from '../shared/package-json';
+import { join as joinPathFragments } from 'path';
```

Comment:
Can we keep this as `join`? It's kind of nice to have some kind of difference between `{ join } from 'path'` and `{ joinPathFragments } from '@nrwl/devkit'`